compare-oracle-to-fallback/src/agent.spec.js

```diff
@@ -107,34 +109,30 @@ describe('Aave oracle versus fallback oracle agent', () => {
       // create a timestamp for when the last alert was triggered, less than 24 hours ago
       const tStart = blockTimestamp - ALERT_MINIMUM_INTERVAL_SECONDS + 1;
 
+      // set the return values for the mocked oracles and token alert time since last triggered
+      initializeData.tokenContractFallbackAlertTuples[0].aaveOracleContractInstance.getAssetPrice
+        .mockResolvedValueOnce(assetPriceOracle);
+      initializeData.tokenContractFallbackAlertTuples[0].fallbackOracleContractInstance
+        .getAssetPrice
+        .mockResolvedValueOnce(assetPriceFallback);
+      initializeData.tokenContractFallbackAlertTuples[0].tokenAlert.tStart = tStart;
+
       // need to create blockEvent (with .block.timestamp and .blockNumber)
       const mockedBlockEvent = createBlockEvent({
         blockNumber: 12345,
-        block: {
-          timestamp: blockTimestamp,
-        },
+        timestamp: blockTimestamp,
       });
 
-      // create the mocked promise to pass in to provideHandleBlock
-      const tokensContractsAlerts = await mockTokensContractsAlertsPromise(
-        assetPriceOracle,
-        assetPriceFallback,
-        tStart,
-      );
-
-      // create the block handler
-      handleBlock = provideHandleBlock(tokensContractsAlerts);
-
       // we expect no alerts because the previous alert occurred within the last 24 hours
       expect(await handleBlock(mockedBlockEvent)).toStrictEqual([]);
 
       // we also expect that the variable tracking the number of alerts in the last 24 hours will
       // be incremented by 1
-      const { tokenAlert } = tokensContractsAlerts[0];
+      const { tokenAlert } = initializeData.tokenContractFallbackAlertTuples[0];
       expect(tokenAlert.numAlertsInLastDay).toStrictEqual(1);
     });
 
-    it('returns no findings if percent error is <=  2%', async () => {
+    it('returns no findings if percent error is <= 2%', async () => {
```

Comment:
As in the other test case, manually set the threshold rather than relying upon what is imported from the configuration file.  Then update this description string as well.
```suggestion
    it('returns no findings if percent error is below the threshold', async () => {
```